doc/compiled.json

```diff
@@ -66,6 +66,9 @@
     {
       "name": "Documents"
     },
+    {
+      "name": "Figma Attachments"
```

Comment:
I added this new group since I saw its just at the bottom of the toc list but I think it should be grouped and sorted correctly. I can also extract it from this PR and create a seperate pr :slightly_smiling_face: